src/app/cli/src/explore/api_server.rs

```diff
@@ -191,7 +191,9 @@ impl APIServer {
     pub async fn pre_run(&self) -> Result<(), InternalError> {
         self.task_executor.pre_run().await?;
         self.flow_executor.pre_run(self.time_source.now()).await?;
-        self.outbox_processor.pre_run().await?;
+        // NOTE: We don't need `self.outbox_executor.pre_run()` here,
+        //       since it has already been made.
+
```

Comment:
Then maybe TE and FE `pre_run` would move to `initialize_server_components` ?

Reply:
That's a good idea!

Reply:
Done